src/program_executor.cpp

```diff
@@ -144,6 +144,17 @@ void ProgramExecutionServer::Execute(
       return;
     }
   }
+
+  moveit_msgs::CollisionObject surface;
+  surface.header.frame_id = robot_config_.base_link();
+  surface.id = "surface_segmentation_collision_table";
```

Comment:
Can you add "surface_segmentation_collision_table" as a constant named `kCollisionSurfaceName` to a file called `motion_planning_constants.h` and use the constant instead? It will be similar to [db_names.h](https://github.com/jstnhuang/rapid_pbd/blob/indigo-devel/include/rapid_pbd/db_names.h).